Src/AutoFixture.xUnit.net2.UnitTest/AutoFixture.xUnit.net2.UnitTest.csproj

```diff
@@ -93,6 +94,9 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
This line jumped out to me. I'm not sure whether it was intentional or not, but it seems it was [automatically added by Visual Studio 2013](https://connect.microsoft.com/VisualStudio/feedback/details/800245/vs2013rc-adds-to-vs2012-c-project-section-itemgroup) to tell Test Explorer that this project contains unit tests.

Either way, I don't think this line is related to the rest of the patch in this commit, so it should either be moved to its own commit to explain why it's there or maybe removed altogether. What do you think?